.travis.yml

```diff
@@ -30,17 +31,23 @@ jobs:
       install: "pip install flake8==3.6.0" 
       script: "flake8"
       after_success: travis_terminate 0
-    - stage: 'Integration Tests'
+    - &integrationtest
+      stage: 'Integration Tests'
       merge_mode: replace
       env: SDK=python
       cache: false
       language: minimal
       install: skip
       before_script:
-        - mkdir $HOME/travisci-tools && pushd $HOME/travisci-tools && git init && git pull https://$CI_USER_TOKEN@github.com/optimizely/travisci-tools.git && popd
+        - mkdir $HOME/travisci-tools && pushd $HOME/travisci-tools && git init && git pull https://$CI_USER_TOKEN@github.com/optimizely/travisci-tools.git sohail/refactorscript && popd
       script:
-        - "$HOME/travisci-tools/fsc-trigger/trigger_fullstack-sdk-compat.sh"
+        - "$HOME/travisci-tools/trigger-script-with-status-update.sh"
       after_success: travis_terminate 0
+    - <<: *integrationtest
+      stage: 'Benchmarking Tests'
+      env: SDK=python REPO_SLUG=Benchmarking
+      script:
+        - "$HOME/travisci-tools/trigger-script-with-status-update.sh yasir/bench-exp-support"
```

Comment:
the yasir/bench-exp-support is for testing right

Reply:
yup

Reply:
We can remove the custom branch and merge in now right?

Reply:
yes, about to remove.